packages/core/contracts/insured-bridge/BridgePool.sol

```diff
@@ -35,7 +35,7 @@ contract BridgePool is Testable, BridgePoolInterface, ExpandedERC20, MultiCaller
     IERC20 public override l1Token;
 
     // Track the total number of relays and uniquely identifies relays.
-    uint256 public numberOfRelays;
+    uint32 public numberOfRelays;
```

Comment:
what? you dont think we are going to have `2^(256)-1` relays?

Reply:
:)